### ISSUE TITLE
Add "-RunImmediate" switch support to the Add-DatabricksNotebookJob cmdlet

### ISSUE THREAD

**Issue by RomainLasso** [DataThirstLtd/azure.databricks.cicd.tools#63]
Hi,

It would be nice if the "**Add-DatabricksNotebookJob**" cmdlet could have the -RunImmediate switch, such as the "**Add-DatabricksPythonJob**" cmdlet.

Best Regards,

Romain

**Comment by simondmorias** [comment]
Created branch 63 for this. However I keep getting an error response from the API:
```
{"error_code":"INVALID_PARAMETER_VALUE","message":"Run settings must be specified."}
```
I'm not sure why - I will have to look into this further or contact Databricks.

**Comment by RomainLasso** [comment]
I don't think that the property '**job_id**' and '**new_settings**' are needed. 

You could just set the $JobBody variable to the $BodyText :

``` ps
    $BodyText = $JobBody | ConvertTo-Json -Depth 10
```
instead of :
``` ps
  $Body = @{}
  $Body['job_id']= $JobId
  $Body['new_settings'] = $JobBody
  $BodyText = $Body | ConvertTo-Json -Depth 10
```

**Comment by simondmorias** [comment]
Well spotted - yes I had an existing job with the same name. Released to PS Gallery as version 1.1.20.
Thanks

**Comment by RomainLasso** [comment]
Thanks
